### ISSUE TITLE
"Select Left" / "Select Right" popups not selectable anymore for single-line changes

### ISSUE THREAD

**Issue by hybridherbst** [ForkIssues/TrackerWin#1924]
So for a few versions only half their area actually could be clicked (the button would disappear on the rest), but seems now they're not clickable at all anymore:

https://github.com/fork-dev/TrackerWin/assets/2693840/f339f5db-22f0-4026-88a9-1d61aeac442f

**Comment by DanPristupov** [comment]
Yes, it happens when there is a conflict in the very first line of file.

**Comment by hybridherbst** [comment]
I see! And then when it's at another part of the file I probably get the half-selectable-only "Select" button

**Comment by gaspo** [comment]
I just wanted to report this, but luckily i checked existing issues first :) This little thing is annoying me for a long time. The button is hiding from me when trying to click unless i click on the red area. 
![image](https://github.com/fork-dev/TrackerWin/assets/353728/ffdb25ab-435a-4079-960b-2ca5a891c774)

**Comment by DanPristupov** [comment]
@gaspo what Fork version do you use?

**Comment by gaspo** [comment]
Hi, sorry i didnt mention it. 1.90.1.0 I guess its the latest. Still happening there.

**Comment by DanPristupov** [comment]
Fixed in 1.91

**Comment by jcavalieri** [comment]
@DanPristupov, I'm having this problem with Version 2.65.0
